server/policy_constants.go

```diff
@@ -135,7 +135,7 @@ func ConvertSCRTMVersionToGCEFirmwareVersion(version []byte) (uint32, error) {
 		asciiVersion = append(asciiVersion, b)
 	}
 
-	versionNum, err := strconv.Atoi(string(asciiVersion))
+	versionNum, err := strconv.ParseUint(string(asciiVersion), 10, 32)
 	if err != nil {
 		return 0, fmt.Errorf("when parsing GCE firmware version: %w", err)
 	}
```

Comment:
You can remove the unnecessary cast to uint32 in the line below this

Reply:
Atoi returns uint64, so still need to cast it to uint32